scan.rb

```diff
@@ -11,13 +11,29 @@ def get_option(key)
   selected.split('=').last
 end
 
+def write_results(output) 
+  begin
+    puts " Start writing final results"
+    file = File.open("output", "ab+")
```

Comment:
Please remove the quotes, otherwise it'd always output in the file called `output`, not the file referenced in the variable `output`.

Reply:
done